models/profileDiffs.js

```diff
@@ -0,0 +1,27 @@
+const firestore = require("../utils/firestore");
+const profileDiffsModel = firestore.collection("profileDiffs");
+
+/**
+ * Fetches the pending profile diffs
+ * @return {Promise<profileDiffsModel|Array>}
+ */
+const fetchProfileDiffs = async () => {
+  try {
+    const snapshot = await profileDiffsModel.where("approval", "==", "PENDING").get();
+    const profileDiffs = [];
+    snapshot.forEach((doc) => {
```

Comment:
Are we handling situation like undefined here?

Reply:
yes